core/handler/manager_utils.go

```diff
@@ -0,0 +1,74 @@
+// Copyright © 2017 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package handler
+
+import (
+	"context"
+
+	pb "github.com/TheThingsNetwork/ttn/api/handler"
+	pb_lorawan "github.com/TheThingsNetwork/ttn/api/protocol/lorawan"
+	"github.com/TheThingsNetwork/ttn/core/handler/device"
+	"github.com/TheThingsNetwork/ttn/core/types"
+	"github.com/TheThingsNetwork/ttn/utils/errors"
+)
+
+const maxAttr uint8 = 5
+
+//eventSelect select the appropriate event for device is updated/created and check if the event is possible
+func (h *handlerManager) eventSelect(ctx context.Context, dev *device.Device, lorawan *pb_lorawan.Device, appId string) (evt types.EventType, err error) {
```

Comment:
Unnecessary refactoring. This function is only used once. Why take it out of the normal flow?